devlib/target.py

```diff
@@ -1035,6 +1035,18 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def adb_reboot_bootloader(self):
+        adb_command(self.adb_name, 'reboot-bootloader', timeout=30)
+
+    def adb_root(self):
+        adb_command(self.adb_name, 'root', timeout=30)
```

Comment:
That seems interesting... however, I quite don't get the potential side-effect on:
https://github.com/ARM-software/devlib/blob/master/devlib/target.py#L739

@setrofim so you think is it safe to update `self.connected_as_root` when this command succeeds?